Gopkg.lock

```diff
@@ -179,6 +187,7 @@
     "github.com/golang/glog",
     "github.com/kubernetes-csi/drivers/pkg/csi-common",
     "github.com/pborman/uuid",
+    "github.com/pkg/errors",
```

Comment:
we don't really need `errors` pkg. For the use case here, `error` is sufficient.